src/runtime/subscription-linking-flow-test.js

```diff
@@ -31,6 +34,12 @@ describes.realWin('SubscriptionLinkingFlow', (env) => {
 
   const PUBLICATION_ID = 'pub1';
   const REQUEST = {publisherProvidedId: 'ppid'};
+  const MULTI_REQUEST = {
```

Comment:
nit: this looks like it's meant to be immutable, so we should either reset it in before each, or freeze it somehow.